consume-more-stuff/server/routes/items.js

```diff
@@ -61,6 +61,21 @@ router.post("/", (req, res) => {
     });
 });
 
+router.get("/:item", (req, res) => {
+  let {item} =req.params
+   return new Item({title:item})
+  .fetch({
+    withRelated: ["category_id", "condition_id", "itemStatus_id"]
+  })
+  .then((item)=>{
+    const itemRes = item.serialize()
+    return res.send(JSON.stringify(itemRes));
+  })
+  .catch((err)=>{
+    return res.send(404).json({message:`Item not found.`})
+  })
+})
+
```

Comment:
The router will see both:
`/:item` and `/:id` as the  same path. I think only one of them should be there and that they're duplicate routes.